.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           fetch-depth: 0
       - uses: codespell-project/actions-codespell@2391250ab05295bddd51e36a8c6295edb6343b0e
         with:
-          ignore_words_list: datas
+          ignore_words_list: datas re-use
```

Comment:
How about excluding the LICENSE file entirely with:
```
with:
  exclude_file: src/foo
```
taken from https://github.com/codespell-project/actions-codespell

Reply:
it also errored out in other files on the same word